### DISCUSSION TITLE
How to restore additional volumes?

### DISCUSSION THREAD

**Discussion by loeffelpan** [nextcloud/all-in-one#5414] [Questions] [answered]
I would like to use your backup solution for other volumes on my server.
Just wondering if there is a built-in possibility to restore these?
At least when running in rootless mode restoring from host would cause permission issues, as `subuid` and `subgid` is used during backup.

**Comment by szaimen** [comment, marked as answer]
Hi, I fear it is currently not possible to restore additonal volumes via the aio interface. 

However see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#pro-tip-backup-archives-access
